Alphabet-Recognize/camera_recognize.py

```diff
@@ -18,14 +21,15 @@ def detectpicture():
 		print("I think it is \'", alphabet, "\'", sep='')
 		match alphabet:
 			case T:
-				return 1
+				send_message.data[3]=1
```

Comment:
List assignment index out of range. This also happens in line `26`, `28` and `31`.